tempodb/encoding/vparquet4/wal_block.go

```diff
@@ -546,7 +546,7 @@ func (b *walBlock) FindTraceByID(ctx context.Context, id common.ID, opts common.
 		}
 	}
 
-	combiner := trace.NewCombiner(opts.MaxBytes)
+	combiner := trace.NewCombiner(opts.MaxBytes, false)
```

Comment:
I have left this to false. Even if we know it shouldn't ever happen I think it is a good sentinel